### DISCUSSION TITLE
Can someone tell me what kind of agent setup is this ?

### DISCUSSION THREAD

**Discussion by DreamFx0000** [Ylianst/MeshCentral#5542] [General]
![Screenshot_229](https://github.com/Ylianst/MeshCentral/assets/150596502/9022c946-bab6-419c-9150-491b17eaf365)

as soon as this guy opens his agent exe file it is asking some kind of code .. once he enter this code then only agent is installing .. 

can you someone please help me how one can achieve this ?

**Comment by si458** [comment]
this isnt possible, check #655 #4047 and a few others for suggestions
i believe tho you can download the built agent exe from the web ui, then use an external application to password lock the file itself
but sadly meshcentral doesnt do this for you

**Reply by si458** [reply]
try this link then but sadly this isnt possible with meshcentral itself! so someone is using a special application for this
https://github.com/Ylianst/MeshCentral/issues/5166#issuecomment-1567440136

**Reply by dinger1986** [reply]
youll see my comment here https://github.com/Ylianst/MeshCentral/issues/5166#issuecomment-1567454038 ask "the guy" how hes done it and report back, its been asked alot but noones come up with a solution or how its been done, including the devs

**Reply by si458** [reply]
do also share the video/link, then we can have a look/make inquires

**Reply by DreamFx0000** [reply]
sadly he wont give .. thats why I recorded it and asking here :(

**Reply by si458** [reply]
this is the problem, if people arent going to at least help the community, then we devs cant help the community either!

**Reply by dinger1986** [reply]
its been asked alot of times, noones helping with it and noone who is willing to help knows what it is. Id suggest locking this @si458 and adding a description that makes sense so other can find it instead of it being soemthing that is continuously asked due to "someone" having implemented something and not feeding it back to the community
